code/modules/reagents/chemistry/reagents/unique/eigenstasium.dm

```diff
@@ -0,0 +1,104 @@
+ *The status effect slowly send you on a wild ride and replaces you with an alternative reality version of yourself unless you consume eigenstasium/bluespace dust/stabilising agent.
+ *During the process you get really hungry,
+ *Then some of your items slowly start teleport around you,
+ *then alternative versions of yourself are brought in from a different universe and they yell at you.
+ *and finally you yourself get teleported to an alternative universe, and character your playing is replaced with said alternative
+ *Spraying this on lockers allows you to create eigenlinked lockers - see the eigenstate subsystem for using this to create your own links
+*/
+/datum/reagent/eigenstate
+	name = "Eigenstasium"
+	description = "A strange mixture formed from a controlled reaction of bluespace with plasma, that causes localised eigenstate fluxuations within the patient"
+	taste_description = "wiggly cosmic dust."
+	color = "#5020F4"
+	overdose_threshold = 15
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	ph = 3.7
+	impure_chem = /datum/reagent/impurity/eigenswap
+	inverse_chem = null
+	failed_chem = /datum/reagent/bluespace //crashes out
+	chemical_flags = REAGENT_DEAD_PROCESS //So if you die with it in your body, you still get teleported back to the location as a corpse
+	data = list("location_created" = null)//So we retain the target location and creator between reagent instances
+	///The creation point assigned during the reaction
+	var/turf/location_created
+	///The return point indicator
+	var/obj/effect/overlay/holo_pad_hologram/eigenstate
+	///The point you're returning to after the reagent is removed
+	var/turf/open/location_return = null
+
+/datum/reagent/eigenstate/on_new(list/data)
+	location_created = data["location_created"]
+
+/datum/reagent/eigenstate/expose_mob(mob/living/living_mob, methods, reac_volume, show_message, touch_protection)
+	. = ..()
+	if(!(methods & INGEST))
+		return
+	if(creation_purity > 0.9 && location_created) //Teleports you home if it's pure enough
+		do_sparks(5,FALSE,living_mob)
+		do_teleport(living_mob, location_created, 0, asoundin = 'sound/effects/phasein.ogg')
+		do_sparks(5,FALSE,living_mob)
+
+//Main functions
+/datum/reagent/eigenstate/on_mob_add(mob/living/living_mob, amount)
+	//make hologram at return point
+	eigenstate = new (living_mob.loc)
+	eigenstate.appearance = living_mob.appearance
+	eigenstate.alpha = 170
+	eigenstate.add_atom_colour("#77abff", FIXED_COLOUR_PRIORITY)
+	eigenstate.mouse_opacity = MOUSE_OPACITY_TRANSPARENT//So you can't click on it.
+	eigenstate.layer = FLY_LAYER//Above all the other objects/mobs. Or the vast majority of them.
+	eigenstate.anchored = 1//So space wind cannot drag it.
+	eigenstate.name = "[living_mob.name]'s eigenstate"//If someone decides to right click.
+	eigenstate.set_light(2)	//hologram lighting
+
+	location_return = get_turf(living_mob)	//sets up return point
+	to_chat(living_mob, "<span class='userdanger'>You feel your wavefunction split!</span>")
```

Comment:
These are still weird, I wish these were descriptions of what you would actually physically feel.

Reply:
How's that?